### ISSUE TITLE
Custom fonts not working in Proton (DXVK) v0.6.6-1

### ISSUE THREAD

**Issue by berglh** [flightlessmango/MangoHud#622]
Custom fonts are working with vkcube and showing Vulkan, but are not working in Proton showing DXVK. GPU model is also not shown, but this might be expected in Proton.

Vulkan:
![image](https://user-images.githubusercontent.com/12455338/139572316-395733ba-e56f-4040-a8c8-6841cb043a73.png)

DXVK:
![image](https://user-images.githubusercontent.com/12455338/139572278-47de6fc4-5e4e-410c-bf34-e6fe397df305.png)

**Comment by flightlessmango** [comment]
I would guess that you don't have access to the font and the same thing is happening in #621. When you run proton you're inside a container which prevents you from accessing the wider filesystem, you could try and put the font somewhere inside your home folder

**Comment by berglh** [comment]
> I would guess that you don't have access to the font and the same thing is happening in #621. When you run proton you're inside a container which prevents you from accessing the wider filesystem, you could try and put the font somewhere inside your home folder

I tried as you suggested and tried putting the fonts in with my MangoHud config in `~/.config/MangoHud`, then referenced that inside the config file I made for this game. However, it was not able to access this file. I then used Steam Console and found the folder where the game executable resided in `/home/me/.steam/steam/steamapps/common/Ori\ and\ the\ Will\ of\ the\ Wisps/`. I copied the font to this location and referenced it without any path, and it worked.

![image](https://user-images.githubusercontent.com/12455338/139577739-e785334d-a725-4f82-bde2-b5349c2d16a0.png)

This more or less makes sense. I guess it might be good to make some kind of comment to this effect for the font mapping option. As this is not actually a bug per se, and a feature of Proton, I will close this issue.
